src/test/java/eu/europa/ted/eforms/sdk/analysis/cucumber/SdkValidationSteps.java

```diff
@@ -153,11 +177,16 @@ public void rule_should_have_been_fired(String expectedRule) {
         .isPresent());
   }
 
-  @Then("^I should get (.*) validation errors?$")
-  public void i_should_get_validation_errors(int errorsCount) {
+  @Then("^I should get (.*) SDK validation errors?$")
+  public void i_should_get_sdk_validation_errors(int errorsCount) {
```

Comment:
Is it possible to use syntax like iShouldGetSdkValidationErrors(, as otherwise this is not correct in terms of the Google code style.

Reply:
Snake case is the Cucumber world's convention of choice. And it does look better (for me, at least) when mirroring potentially long sentences (as is the norm with BDD)
e.g., i_should_do_this is more readable than iShouldDoThis

You can do this later on its own PR, as it's a little bit of work to change it everywhere.
Also, you can add this to junit-platform.properties, for the auto-generated example snippets to use camel case:
cucumber.snippet-type=camelcase

Reply:
We will have to see later how we deal with this, maybe for checkstyle we could ignore some rules for this project.